packages/react-core/src/components/ExpandableSection/ExpandableSection.tsx

```diff
@@ -18,7 +18,7 @@ export interface ExpandableSectionProps {
   /** Text that appears in the toggle when collapsed (will override toggleText if both are specified; used for uncontrolled expandable with dynamic toggle text) */
   toggleTextCollapsed?: string;
   /** Callback function to toggle the expandable content */
-  onToggle?: () => void;
+  onToggle?: (isExpanded: boolean) => void;
```

Comment:
In order to not break consumers this argument needs to be optional.
```suggestion
  onToggle?: (isExpanded?: boolean) => void;
```

Reply:
This is correct:
```
 onToggle?: (isExpanded: boolean) => void;
```

You want to specify that `isExpanded` is always provided as a boolean value. By using `isExpanded?` the type becomes `boolean | undefined`. But a boolean value is always provided.

Clients are free to ignore this param as per the previous implementation.

Reply:
Yup I see that.